Formula/semgrep.rb

```diff
@@ -127,7 +127,7 @@ def install
       ENV["PREFIX"] = prefix
 
       system "opam", "init", "--no-setup", "--disable-sandboxing"
-      ENV.deparallelize { system "opam", "switch", "create", "ocaml-base-compiler.4.10.0" }
+      ENV.deparallelize { system "opam", "switch", "create", "ocaml-base-compiler.4.10.2" }
```

Comment:
Would this work?
```suggestion
      ENV.deparallelize { system "opam", "switch", "create", "ocaml-base-compiler.#{Formula["ocaml"].version}" }
```

Reply:
Is that guaranteed to be 4.10.2?

Reply:
It isn't; it's likely to be 4.12 relatively soon.

Reply:
Probably best to leave as 4.10.2 literal for now then?